File: help/pt_pt/exercicios_da_ficha_9/Ficha09_016/Ficha09_016.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Numero : ");
    int num;
    scanf("%d", &num);
    int i = 1 ;
    printf("Divisores : ");
    while(i < num ){
        if(num % i == 0 ){
            printf("%d ",i );
        }
        i++;
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:27

File: help/pt_pt/exercicios_da_ficha_9/Ficha09_016/Ficha09_016.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Numero : " ;
    $num = intval(isset($_POST["num"])? $_POST["num"] : isset($_GET["num"]) ? $_GET["num"] : 0);
    
    echo $num . "<br/>";// print input 
    $i = 1 ;
    echo  "Divisores : " ;
    while($i < $num ){
        if($num % $i == 0 ){
            echo $i  .  " " ;
        }
        $i = $i + 1 ;
    }
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Thu, 2022 Sep 29 12:27

?>